File: apps/web/src/types/voice.ts
export type Voice = {
  id: string
  languageCode: string
  languageShort: string
  voiceType: string
  displayName: string
  resourceId: string
  capturedAt: string | null
}

export type VoiceListResponse = {
  items: Voice[]
  page: number
  pageSize: number
  total: number
}

File: apps/api/app/schemas/voice.py
from pydantic import BaseModel

class VoiceResponse(BaseModel):
    id: str
    languageCode: str
    languageShort: str
    voiceType: str
    displayName: str
    resourceId: str
    capturedAt: str | None = None

class VoiceListResponse(BaseModel):
    items: list[VoiceResponse]
    page: int
    pageSize: int
    total: int
